packages/orbit-design-tokens/src/dictionary/formatters/json-tokens.ts

```diff
@@ -2,13 +2,20 @@ import { Property } from "style-dictionary";
 
 import { getValue } from "../utils/get";
 
+const getDeprecatedToken = deprecatedToken => {
+  if (deprecatedToken) {
+    return deprecatedToken.name;
```

Comment:
Just to confirm: aside from extracting this into a function, this also changed from `deprecatedToken.value.name` to `deprecatedToken.name`, right?

Reply:
Yes, on purpose. We want to always return the name of the token with `deprecatedToken.name`. If the value is aliased, the result would be something from the foundation. I will also implement a bit smarter replacement values, so if there's not any available replacement, the eslint-plugin would also suggest to use some global value.